sympy/solvers/tests/test_ode.py

```diff
@@ -2901,3 +2901,27 @@ def test_issue_14395():
     sol = Eq(f(x), (C1 - x/3 + sin(2*x)/3)*sin(3*x) + (C2 + log(cos(x))
         - 2*log(cos(x)**2)/3 + 2*cos(x)**2/3)*cos(3*x))
     assert dsolve(Derivative(f(x), x, x) + 9*f(x) - sec(x), f(x)) == sol
+
+    
+def test_sysode_linear_neq_order1():
+    Z0 = Function('Z0')
+    Z1 = Function('Z1')
+    Z2 = Function('Z2')
+    Z3 = Function('Z3')
+
```

Comment:
It seems that trailing spaces were added in this file (those will have to be removed). See the failing [job](https://travis-ci.org/sympy/sympy/jobs/435601684).
You can also test the quality of your code changes locally by running `./bin/test quality`.

Reply:
Yeah sorry about that, I tried reading the wiki to be a little better, but some things aren't really explained in there or at least I don't understand some of them. Probably wasn't supposed to upload the file instead of just pushing the change too. As you can see I'm having a hard time following the etiquette you guys have.

Reply:
That's fine for a new contributor. For windows you can run the code quality tests using `python bin/test quality` from the command line.

It seems that the last file you added contains windows line endings, that's why the diff has grown large.